docker-compose.yml

```diff
@@ -23,6 +24,7 @@ services:
         working_dir: /fcc
         command:
             - npm
-            - start
+            - run
+            - develop
```

Comment:
I think this makes sense for development, if other people prefer 'npm start' that's fine too. It should def not do in if you deploy using Docker -I'm pretty sure you are not.

Reply:
It's on the road map to to move completely to docker. But that is some way away. This should be fine.

Reply:
Great. Any idea on the timeline? Thinking of it, you probably would not want to use Docker Compose for orchestration of your deployments anyway.